packages/plugin-cosmos/package.json

```diff
@@ -0,0 +1,30 @@
+{
+    "name": "@elizaos/plugin-cosmos",
+    "version": "0.1.0-alpha.1",
+    "main": "dist/index.js",
+    "type": "module",
+    "types": "dist/index.d.ts",
+    "dependencies": {
+        "@cosmjs/amino": "^0.29.2",
+        "@cosmjs/cosmwasm-stargate": "^0.32.4",
+        "@cosmjs/stargate": "^0.29.2",
+        "@elizaos/core": "workspace:*",
+        "bignumber.js": "9.1.2",
+        "chain-registry": "*",
+        "cosmjs-utils": "*",
+        "node-cache": "5.1.2",
```

Comment:
do we want another caching system? we should unify this with our caching in the future